### ISSUE TITLE
Executive Summary Paper

### ISSUE THREAD

**Issue by CaptainSifff** [the-teachingRSE-project/competencies#364]
We would need an executive summary paper. I think we could publish that with PLOS CompBio if it is below 3000 words, 
And also the Dagstuhl series thing was receptive of this. For Dagstuhl, there was a deadline of Nov. 1.

**Comment by MakisH** [comment]
I totally agree with advertising one piece of work, but reworking the same piece of work into multiple peer-reviewed formats sounds to me against the good scientific practice guidelines ("salami publications") and could be misunderstood as such by some people, even if the intention is dissemination.

I understand we now have/aim for:

- Main manuscript in revisions:
   - The preprint on Arxiv
   - The de-RSE position paper
   - A peer-reviewed publication
- Spin-offs for dissemination:
   - Blog post
- Additional peer-review formats:
   - Shorter summary in Dagstuhl proceedings
   - This executive summary

What is the motivation behind the latter two, and why two different ones? Or were the last two supposed to be the same?

**Comment by CaptainSifff** [comment]
Yes the executive summary and the Dagstuhl seminar series thing are supposed to be the same, at least initially. We have just bumped too often into problems with the length.... therefore the idea to have a shorter version.
